conda/compass_env/spec-file.template

```diff
@@ -8,7 +8,7 @@ cartopy_offlinedata
 cmocean
 esmf=*={{ mpi_prefix }}_*
 ffmpeg
-geometric_features=0.5.0
+geometric_features=0.6.0
```

Comment:
@hollyhan could you mention this version change explicitly at the bottom of he description of the PR?

Reply:
Thanks!

Reply:
Of course. Thanks for the reminder to update the PR description! I've also just added the links to the other two PRs that go parallel to this one.